### PR TITLE
use oc rhel8 client when SNO prov node is running RHEL8

### PR DESCRIPTION
**Pull request by manurodriguez**
Test-Hint: no-check

### CODE REVIEW

**Comment by manurodriguez** [comment]
retest

File: roles/sno_installer/tasks/10_get_oc.yml [hunk 1]
```diff
@@ -148,7 +159,7 @@
 
 - name: "Unarchive client tarball"
   ansible.builtin.unarchive:
-    src: "{{ cache_dir }}/{{ version }}/openshift-client-linux-{{ version }}.tar.gz"
+    src: "{{ cache_dir }}/{{ version }}/{{ installer_oc_installer_prefix }}.tar.gz"
```
Comment (tonyskapunk) [inline roles/sno_installer/tasks/10_get_oc.yml:162]:
```suggestion
    src: "{{ cache_dir }}/{{ version }}/{{ installer_oc_installer_prefix }}-{{ version }}.tar.gz"
```

Comment (manurodriguez) [reply roles/sno_installer/tasks/10_get_oc.yml]: good catch, thanks

**Comment by manurodriguez** [comment]
Previous job with 4.16 passed, now testing previous OCP versions are not impacted

**Comment by manurodriguez** [comment]
retest

**Comment by manurodriguez** [comment]
SNO 4.15 job passed too, ready for review!

**Review by tonyskapunk — APPROVED** [review]
LGTM, thanks @manurodriguez

**Pull request merged**